content/react/concepts/react-native/react-native.md

```diff
@@ -16,9 +17,9 @@ CatalogContent:
 
 <link rel="canonical" href="https://www.codecademy.com/resources/blog/what-is-react-native/" />
 
-[React Native](https://reactnative.dev/) is an open-source mobile development framework that works across different operating systems like iOS and Android. It was released in 2015 by Facebook (now Meta) and can be used on many smart devices, such as phones, personal computers, and TVs, to provide a consistent interface and user experience.
+[**React Native**](https://reactnative.dev/) is an [open-source](https://www.codecademy.com/resources/docs/open-source) mobile development [framework](https://www.codecademy.com/resources/docs/general/framework) that works across different operating systems like iOS and Android. It was released in 2015 by Facebook (now Meta) and can be used on many smart devices, such as phones, personal computers, and TVs, to provide a consistent interface and user experience.
 
-Traditional mobile app development required multiple codebases for each operating system (iOS, Android, AppleTV, etc.) along with their associated programming languages ([Swift](https://www.codecademy.com/resources/docs/swift), [Java](https://www.codecademy.com/resources/docs/java), etc.). However, React Native provides a single codebase written in [React](https://www.codecademy.com/resources/docs/react)-based JavaScript and rendered in the device's native language.
+Traditional mobile app development required multiple codebases for each operating system (iOS, Android, AppleTV, etc.) along with their associated programming languages such as [Swift](https://www.codecademy.com/resources/docs/swift) and [Java](https://www.codecademy.com/resources/docs/java). However, React Native provides a single codebase written in [React](https://www.codecademy.com/resources/docs/react)-based [JavaScript](https://www.codecademy.com/resources/docs/javascript) and rendered in the device's native language.
```

Comment:
Google recommends that Android apps should now be written in Kotlin as opposed to Java.